src/icons/status/gen_status.sh

```diff
@@ -0,0 +1,26 @@
+#/usr/bin/env bash
+
+png () {
+  middle=$(($3 / 2))
+  center=$((middle - 1))
+
+  convert -size "$3x$3" xc:transparent \
```

Comment:
very cool

Reply:
not for this PR, but would be good to document the dependencies for running this script.

Reply:
Will do later :) I can even just add a comment there or add to the README. There's something I don't really like about that: not sure if the sizes are perfect. They look good but it seems that they're not as correct as they should be. But we can take a look at that later.